HerPublicWebsite.BusinessLogic/ExternalServices/EpbEpc/EpbEpcApi.cs

```diff
@@ -1,11 +1,80 @@
-﻿namespace HerPublicWebsite.BusinessLogic.ExternalServices.EpbEpc
+﻿using HerPublicWebsite.BusinessLogic.Models.Enums;
+using HerPublicWebsite.BusinessLogic.ExternalServices.Common;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using System.Net.Http.Headers;
+
+namespace HerPublicWebsite.BusinessLogic.ExternalServices.EpbEpc
 {
+
+    public record struct EpcAddress
```

Comment:
Could you please add a `Dto` suffix to these struct names so it's clear that they are coming from the EPC API.
We may want to add a class to the Models to hold the information we actually want for the site (e.g. I don't think we'll actually need the address)